src/sentry/models/outbox.py

```diff
@@ -249,13 +252,18 @@ def infer_identifiers(
                     shard_identifier = model.id
                 elif hasattr(model, "organization_id"):
                     shard_identifier = model.organization_id
-                elif hasattr(model, "auth_provider_id"):
-                    shard_identifier = model.auth_provider_id
+                elif hasattr(model, "auth_provider"):
+                    shard_identifier = model.auth_provider.organization_id
```

Comment:
Is `auth_provider_id` entirely unused now? I don't see it leaving anywhere else in this diff.

Reply:
No, but the shard identifier should be the related organization_id and not the auth_provider_id itself.  This was a minor bug I noticed along the way.  It doesn't have much impact currently in practice.